### ISSUE TITLE
Copy/paste of attachments not working.

### ISSUE THREAD

**Issue by xoudini** [xoudini/emojica#4]

**Comment by Rashesh-Bosamiya** [comment]
I have some emojica images in my UITextView.

Now, I am going to copy those emojis.
The problem happens when attempt to paste those emojis.

Let me know if this doesn't make sense.

**Comment by xoudini** [comment]
Yes, I understand. At the moment you'll have to revert the text manually before allowing the `UITextView` to [copy the selection](https://developer.apple.com/library/content/documentation/StringsTextFonts/Conceptual/TextAndWebiPhoneOS/UsingCopy,Cut,andPasteOperations/UsingCopy,Cut,andPasteOperations.html#//apple_ref/doc/uid/TP40009542-CH11-SW22) to `UIPasteboard`.

I'll mark this as a feature request, and will include this functionality into the next version.
